src/components/app/navigation.styles.scss

```
@@ -22,35 +81,52 @@ $unreadBubbleAnimation: count 500ms ease 500ms forwards;
 				flex-direction: column;
 				min-height: 0;
 				overflow-x: visible;
-				overflow-y: auto;
+				overflow-y: hidden;
 
 				.navigation__itemContainer {
 					align-items: center;
 					width: 100%;
 					padding: 0;
```

Stylelint (17.14.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)